default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchFromGitHub, openssl, stdenv, libpulseaudio, pkg-config, protobuf, lame, libopus }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rfbproxy";
+  version = "0.1.1-90b68e1";
```

Comment:
@cbrewster do you know if there's a way for the nix builder to use the cargo version as-is? we'll have to add a CI thingy to automatically bump the version on every commit or something. otherwise this version won't reflect reality!

Reply:
The versioning here will be tricky, we could add the build version as a parameter and then have our pid1 derivation pass in the pinned revision from niv. Not sure if there is a better solution :disappointed:.